### PR TITLE
Added Utils plugin

### PR DESCRIPTION
**Pull request by bobmulder**
Added Utils plugin (yeah @jadb, we seperated the plugin ;)), and changed CakeManager's description.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -325,13 +325,14 @@ Focus is on the specific task.
 *Misc plugins and libraries*
 
 - [Ajax plugin](https://github.com/dereuromark/cakephp-ajax) - A plugin to ease handling AJAX requests.
-- [CakeManager plugin](https://github.com/cakemanager/cakephp-cakemanager) - Manage your app: User-Management, built in admin-area and many components and behaviors available for your app.
+- [CakeManager plugin](https://github.com/cakemanager/cakephp-cakemanager) - Manage your app: User-Management and a built in admin-area. Ready for use :)
```
Comment (jadb) [inline README.md]: How about:

```
A user management plugin with a built-in admin area.
```

I don't think `Ready for use :)` is necessary as it's the 'awesome' list, so all listed plugins are supposed to be ready for use ;)

File: README.md [hunk 2]
```diff
@@ -325,13 +325,14 @@ Focus is on the specific task.
...
 - [Dashboard plugin](https://github.com/gourmet/dashboard) - Build beautiful dashboards for your cakes!
 - [Sequence plugin](https://github.com/ADmad/cakephp-sequence) - Behavior for maintaining ordered list of records.
 - [Setup:Maintenance](https://github.com/dereuromark/cakephp-setup/blob/cake3/docs/Maintenance/Maintenance.md) - Maintenance shell to go into maintenance mode for all requests with optional IP whitelisting.
 - [Tools plugin](https://github.com/dereuromark/cakephp-tools) - Containing lots of useful libs, helpers, behaviors, components, shells, ...
 - [Travis](https://github.com/FriendsOfCake/travis) - Easy travis setup for CakePHP plugins.
 - [UserTools plugin](https://github.com/burzum/cakephp-user-tools) - User tools for login, registration, password reset and more. Works out of the box CRUD like and is highly configurable.
+- [Utils plugin](https://github.com/cakemanager/cakephp-utils) - Containing useful components, behaviors, helpers and shells.
```
Comment (jadb) [inline README.md]: How about listing some of the stuff it includes? Makes it easier for people to find it (i.e. looking for `slug` for example).

File: README.md (hunk 1 above)
Comment (bobmulder) [inline README.md]: Haha what I ment to say is that it's easy to install and you can use it directly ;) but yeah, you are right ;)

File: README.md (hunk 2 above)
Comment (bobmulder) [inline README.md]: I don't know how many space I have, but I'll add some ;)

**Comment by bobmulder** [comment]
@jadb: added description about components and behaviors

**Pull request merged**